src/components/MyInfo/Contents/BusApply/style.ts

```diff
@@ -39,6 +39,26 @@ export const BusApplyButton = styled.button<{ isDisable: boolean }>`
           color: white;
         `}
 `;
+export const BusApplyDelete = styled.button<{ isDisable: boolean }>`
```

Comment:
```suggestion
export const BusApplyCancelButton = styled.button<{ isDisable: boolean }>`
```
Button이라고 명시해주는게 좋아보입니다.

Reply:
넵 변경하겠습니다